src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindow.cs

```diff
@@ -115,10 +115,11 @@ private void CompositeGUI()
 
                     RenderTitle(title, Color.green);
                 }
+
                 EditorGUILayout.BeginHorizontal("Box");
                 {
-                    string[] compositionOptions = new string[] { "Final composited texture", "All intermediate textures" };
-                    GUIContent renderingModeLabel = new GUIContent("Display texture", "Choose between displaying the composited video texture or seeing intermediate textures displayed in 4 sections (top right: input video, bottom right: alpha mask, bottom left: opaque hologram, top left: final alpha-blended hologram)");
+                    string[] compositionOptions = new string[] { "Final composited texture", "Quadrant with intermediate textures" };
+                    GUIContent renderingModeLabel = new GUIContent("Display texture", "Choose between displaying the composited video texture or seeing intermediate textures displayed in 4 sections (bottom left: input video, top left: opaque hologram, top right: hologram alpha mask, bottom right: hologram alpha-blended onto video)");
```

Comment:
Can quadrant compositing be enabled even if the hardware doesn't support quadrants? Some folks may want to see intermediate textures even if their capture cards don't support 4K?

Reply:
The quadrant view is never output to the capture card, only to the video encoder that writes videos to disk. I believe 4K output on the capture card requires 4K input from the capture card, and neither of those scenarios are currently supported by the compositor